spec/spec_helper.rb

```diff
@@ -57,7 +61,11 @@ class Application < Rails::Application
   # drop the database after the suite.
   config.after(:suite) do
     if ENV["CI"]
-      Mongoid::Threaded.sessions[:default].drop
+      if Mongoid.respond_to?(:default_client)
```

Comment:
I actually don't even know if this line is still required, but yes mongoid-compatibility may prove useful.
I'm using my  branch at the moment, so tell me if this needs improvements or can be merged soon ;)